modules/troubleshooting-network-observability-loki-resource-exhausted.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+
+// * networking/network_observability/troubleshooting-network-observability.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-troubleshooting-loki-resource-exhausted_{context}"]
+= Troubleshooting Loki ResourceExhausted error
+Loki may return a "ResourceExhausted" error when network flow data sent by Network Observability exceeds the configured maximum message size. When using the Loki operator, this maximum message size is configured to 100 MiB. 
+
+.Procedure
+. Navigate to *Operators* -> *Installed Operators*, viewing *All projects* from the *Project* dropdown.
+. Look under the *Provided APIs* heading for the Network Observability Operator, select *Flow Collector* then the *YAML view* tab. 
+.. If you are using the Loki Operator, check that the `spec.loki.batchSize` value does not exceed or come close to 100 MiB. You must keep a margin such as 98 MiB max.
```

Comment:
```suggestion
.. If you are using the Loki Operator, check that the `spec.loki.batchSize` value does not exceed 98 MiB.
```

Suggested editing for minimalism - it's better to tell users what the limit is here clearly, IMO.

Reply:
I can definitely see this. Great suggestion!